dev/linux/integration.sh

```diff
@@ -5,8 +5,8 @@ set -o errtrace -o pipefail -o errexit
 TEST_SPLITS="${TEST_SPLITS:-1}"
 TEST_GROUP="${TEST_GROUP:-1}"
 
-eval "$(sudo /opt/conda/bin/python -m conda init --dev bash)"
 sudo su root -c "/opt/conda/bin/conda install -yq conda-build"
```

Comment:
is this better than `sudo /opt/conda/bin/conda install -yq conda-build`

Reply:
No idea, sought to keep the fix as small as possible as a larger overhaul is needed soon anyhow.

Reply:
I honestly think most of these quirks can be removed without any ill effects